src/OrbitCore/Path.cpp

```diff
@@ -60,13 +69,31 @@ std::filesystem::path Path::CreateOrGetOrbitAppDataDir() {
   return path;
 }
 
-std::filesystem::path Path::GetLogFilePathAndCreateDir() {
+[[nodiscard]] uint32_t Path::GetCurrentProcessId() {
+#ifdef _WIN32
+  uint32_t current_pid = static_cast<uint32_t>(::GetCurrentProcessId());
+#else
+  uint32_t current_pid = static_cast<uint32_t>(getpid());
+#endif
+  return current_pid;
+}
+
+std::filesystem::path Path::CreateOrGetLogDir() {
   std::filesystem::path logs_dir;
   if (!absl::GetFlag(FLAGS_log_dir).empty()) {
     logs_dir = absl::GetFlag(FLAGS_log_dir);
   } else {
     logs_dir = Path::CreateOrGetOrbitAppDataDir() / "logs";
   }
   std::filesystem::create_directory(logs_dir);
-  return logs_dir / "Orbit.log";
+  return logs_dir;
+}
+
+std::filesystem::path Path::GetLogFilePath() {
+  std::filesystem::path logs_dir = CreateOrGetLogDir();
+  std::string timestamp_string =
+      absl::FormatTime(kLogFileNameTimeFormat, absl::Now(), absl::LocalTimeZone());
+  std::string filename =
+      absl::StrFormat(kLogFileNameDelimiter, timestamp_string, Path::GetCurrentProcessId());
+  return logs_dir / filename;
```

Comment:
return CreateOrGetLogDir() / GetLogFilename();
GetLogFilename function will be in OrbitBase.

Reply:
Done! Thanks so much for the suggestion!